src/hooks/useInitialValues.test.ts

```diff
@@ -333,4 +334,228 @@ describe('useInitialValues', () => {
     expect(allFormFields.find((field) => field.id === 'testOrder_1')).not.toBeNull();
     expect(allFormFields.find((field) => field.id === 'testOrder_2')).not.toBeNull();
   });
+
+  it('should return syncronous calculated values for calculated fields in "edit" mode', async () => {
+    let hook = null;
+    let syncronousCalculateExpressionFields: Array<FormField> = [
```

Comment:
You have a typo in the word "synchronous" but since the name of the unit test already reflects the fact that we are dealing with synchronous expressions, I would simply use: `formFields`